roles/ovirt-provider-ovn-driver/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+- block:
+  - name: Install ovirt-provider-ovn-driver
```

Comment:
Can you please do a new line between each task? We do it in all of other roles, and I think it looks more readable. Thanks.

Reply:
Done.